src/components/Exchange/ConfirmationBox.js

```diff
@@ -135,6 +143,35 @@ export default function ConfirmationBox(props) {
     }
   }, [orders, chainId, isLong, toToken.address, toToken.isNative]);
 
+  const existingTriggerOrdersThatWillBeClosed = useMemo(() => {
+    const wrappedToken = getWrappedToken(chainId);
+    return orders.filter((order) => {
+      if (order.type !== DECREASE) return false;
+      const sameToken =
+        order.indexToken === wrappedToken.address ? toToken.isNative : order.indexToken === toToken.address;
+      if (order.isLong !== isLong || !sameToken) {
+        return false;
+      }
+      let canImmediatelyClosed;
```

Comment:
maybe `willImmediatelyClose`?

Reply:
or better use "execute" because order will not necessarily close position